backend/routers/slack.router.js

```diff
@@ -2,23 +2,26 @@ const express = require("express");
 const router = express.Router();
 const { App } = require("@slack/bolt");
 const cron = require("node-cron");
-const { Event } = require("../models/event.model");
-const { Project } = require("../models/project.model");
+const Event = require("../models/event.model");
+const Project = require("../models/project.model");
 
 //https://api.slack.com/web
 
 const app = new App({
-  token: "xoxb-1273107934163-1285590347713-T4jPga74wL2iddE1J1r5w6a0",
-  signingSecret: "4c1f11f5986ab05b95a796b7897e87c3",
+  token: process.env.SLACK_BOT_TOKEN,
+  signingSecret: process.env.SLACK_SIGNING_SECRET,
 });
 
 //Checks DB every monday (1) for slack messages to schedule this week
 // cron.schedule("* * * * 1", () => {});
 
-(async () => {
-  await app.start(3001);
-  console.log("Connected to Slack");
-})();
+// TODO: Refactor this server out of the router. This server instance is breaking the tests.
+if (process.env.NODE_ENV !== "test") {
+  (async () => {
+    await app.start(3002);
+    console.log("Connected to Slack");
+  })();
+}
```

Comment:
The comment still stands. There is a lot of issues having the Slack Node app be instantiated in the routes. When the tests run, it sets up the application. If we don't change the development environment to 'test' specifically, the Slack application instantiates as is seen here. This is breaking our tests. 

I have talked to Alex, Jonathan, and Susan about this. This is out of scope for this project, but we need to review our Slack story and find a way to move that server out of our application pathway. Or some other methods other than explicitly setting the node environment, so that we can have flexibility on what node_environment we want to use.